.github/workflows/integration-tests.yml

```diff
@@ -23,3 +23,9 @@ jobs:
       run: bash -ci "$(curl -fsSL https://raw.githubusercontent.com/onflow/flow-cli/master/install.sh)" -- v1.3.2
     - name: Run tests
       run: sh ./test.sh
+    - name: Normalize coverage report filepaths
+      run : sh ./normalize_coverage_report.sh
```

Comment:
Should this script be checked in somewhere?

Reply:
Realized I left that script in the action and just removed it. Having trouble what's going on in [that script](https://github.com/m-Peter/flow-code-coverage/blob/main/normalize_coverage_report.sh), but some manipulation of the report is necessary since the last run failed in Codecov on `unusable report`

Reply:
Fixed, but there's no diff to compare to since this is the first PR with the action.

Reply:
@sisyphusSmiling The `LCOV` format needs actual file paths, which as of now, it is not yet possible to generate in the code coverage report. This should be fixed with the addition of a `sourceFile()` pragma: https://github.com/onflow/flow-emulator/pull/437. After I integrate this in the code coverage feature, no normalization/mapping to file paths will be required.

Reply:
I would also suggest a change to the CI workflow, as I have done here:
```yml
- name: Install Flow CLI
  run: bash -ci "$(curl -fsSL https://raw.githubusercontent.com/onflow/flow-cli/master/install.sh)"
- name: Run tests
  run: flow test --cover --covercode="contracts" --coverprofile="coverage.lcov" tests/test_*.cdc
```
Using the latest available `Flow CLI` version, and utilizing the `--covercode="contracts"` flag, which filters out code coverage for scripts & transactions.